### ISSUE TITLE
CSS Parser Error

### ISSUE THREAD

**Issue by ccalamos** [jhildenbiddle/css-vars-ponyfill#97]
I noticed that the parser believes that nested elements such within the print page media query is invalid.

For instance

```css
@media print {
 @page {
  size: A4;
  margin: 10%;

  @bottom-left-corner {
    content: "Page " counter(page);
  }
 }
}
```

is valid CSS, however the parser throws a  Expected '}' error.

**Comment by jhildenbiddle** [comment]
Thanks @ccalamos. I'll take a look.

**Comment by ManikandanGovindarajan** [comment]
@jhildenbiddle any update on this bug please.

**Comment by jhildenbiddle** [comment]
Apologies for the slow response, @ccalamos and @ManikandanGovindarajan. I recently relocated which has been keeping me busy.

It's looking like next week is the earliest I'll be able to dive into this. If you want to have a go, PRs welcome!

In the meantime, you can avoid the parsing error by moving the associated CSS block into a separate `<style>` or `<link>` element, then use [options.exclude](https://jhildenbiddle.github.io/css-vars-ponyfill/#/?id=exclude) to ignore it. It's not ideal, but perhaps it will help until the bug can be fixed.

**Comment by jhildenbiddle** [comment]
Fixed in 2.2.1
